### ISSUE TITLE
Accessing blocked IPs

### ISSUE THREAD

**Issue by mikeruss1** [CIDRAM/CIDRAM#314]
wondering if there is an easy way of accessing the list of banned IPs ?

There are normally around 20, which probably have a noticeable traffic rate. It would be easy for me to block them as I have a short piece of code that generates 429s, which runs before CIDRAM, and it would save on loading CIDRAM code and tables.

maybe a possibility for V3?

**Comment by Maikuolan** [comment]
I'll see what I can do. :-)

Not sure yet how easy/difficult it'll be, because I'm currently in the process of completely rewriting how IP tracking works for v3, and I'm not 100% sure how it'll look by the time I've finished the rewrite. Should be doable though.

**Comment by Maikuolan** [comment]
Just to clarify: We're talking about fetching a list of banned IPs directly through code, right? So that you can leverage it with some external code/script/etc? Or do you mean from the front-end or something similar?

**Comment by mikeruss1** [comment]
Yes code, the API extends to various requests, one could be return a list of banned IPs, which can be leveraged externally to cidram.
